assets/product-images-slider.js

```diff
@@ -0,0 +1,62 @@
+const thumbnailSlides = document.querySelector('[data-glide-thumbnail-slides]').dataset.glideThumbnailSlides
```

Comment:
Add an early return after this, so that you avoid any errors if the querySelector doens't find anything.
You should also add conditional chainning to your properties chain here for additional saftey.

Reply:
When I try to create an early return here, I get an error that says that it is an illegal return statement.  

`const numberOfThumbnailSlides = document.querySelector('[data-glide-thumbnail-slides]')?.dataset?.glideThumbnailSlides
if (numberOfThumbnailSlides === undefined) return`
<img width="403" alt="Screen Shot 2023-02-09 at 2 31 40 PM" src="https://user-images.githubusercontent.com/92998577/217932117-012711af-eb5d-44f5-a28c-b66d8a385aa5.png">